docs/client-side/yeoman.rst

```diff
@@ -0,0 +1,260 @@
+You can add the Yeoman generated files to a Visual Studio project. From there, the Visual Studio project **Dependencies** node lets you restore packages from the UI instead of the command line. 
+
+1.	In the Visual Studio main menu, select :menuselection:`File --> New --> Project...`
+
+2.	In the New Project dialog, select the ``ASP.NET Web Application`` web template and click the ``OK`` button.
+
+3.	In the New ASP.NET Project dialog, select the ``ASP.NET 5 Empty`` template. Click the ``OK`` button to create the project. At this point you have a minimal project that includes a **Dependencies** node and a **project.json** file. 
+
+.. image:: yeoman/_static/yeoman-yowebappvs-solution.png   
+ 
+4.	From the directory containing the generated files, copy the entire ``/app`` directory, along with ``bower.json``, ``gulp.js`` and ``package.json`` files, to the Visual Studio project directory. The project should now look something like the screenshot below.
+ 
+.. image:: yeoman/_static/yeoman-yowebappvs-copied-files.png   
+
+5.	In the Solution Explorer, open the ``Dependencies`` node, right-click the ``Bower`` node  and select ``Restore Packages`` from the context menu.
+ 
+6.	Right-click the ``NPM`` node  and select ``Restore Packages`` from the context menu. This may take a little time to complete. 
+
+.. note:: Why restore packages a second time for NPM and Bower? You could just copy the the node_modules and bower_components directories into your project, but the NPM and Bower versions used by Visual Studio may differ from the global command line version. 
+
+7.	Click the ``Task Runner Explorer`` refresh button. 
+
+.. image:: yeoman/_static/yeoman-task-runner-refresh.png   
+ 
+8.	A list of tasks will appear in the Task Runner Explorer. Double-click the ``serve`` task. The **serve** task will launch the browser again and watch for file changes. 
+
+9.	Open ``/app/Index.html`` and change the heading "Running as a Visual Studio Web Site" to "Running as a Visual Studio Project" and save the file. The changes will appear automatically in the browser. 
+
+.. image:: yeoman/_static/yeoman-browser-running-vsproj.png   
+ 
+10.	In the Task Runner Explorer, double-click the ``build`` task. The task will populate the **/dist** folder. 
+
+Setting the Web Root Folder
+===========================
+
+How do you push the entire web site out to the **wwwroot** directory? The Gulp **build** task populates a folder called **/dist**. You could change all **/dist** references in gulpfile.js to **/wwwroot**. Another approach is to configure the project to recognize **/dist** as the root folder instead of wwwroot. To do this, open ``project.json`` and change ``"webroot": "wwwroot"`` to ``"webroot": "dist"``. Once you save the file, Visual Studio will recognize **dist** as the root of the web application and the change will be reflected in Solution Explorer.
+
+.. image:: yeoman/_static/yeoman-webroot-dist.png   
+ 
+Generating a Chrome Plugin
```

Comment:
I think generating a Chrome Plugin is outside the scope of what we should be documenting here - let's remove this content.